airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -141,14 +144,24 @@ public JsonNode toJdbcConfig(final JsonNode config) {
 
   @Override
   protected JdbcSqlGenerator getSqlGenerator(final JsonNode config) {
-    throw new UnsupportedOperationException("mysql does not yet support DV2");
+    return new MysqlSqlGenerator();
   }
 
   @Override
   protected StreamAwareDataTransformer getDataTransformer(ParsedCatalog parsedCatalog, String defaultNamespace) {
     return new PropertyNameSimplifyingDataTransformer();
   }
 
+  @Override
+  public boolean isV2Destination() {
+    return true;
+  }
+
+  @Override
+  protected boolean shouldAlwaysDisableTypeDedupe() {
+    return true;
```

Comment:
We don't ever type and dedupe in MySQL?

Reply:
I removed this in the next pr https://github.com/airbytehq/airbyte/pull/36936/files#diff-b6b4eb9627731eeb01e421a303a382207ac09a3b3d4408bec78492303be9f502L161. Just wanted to split the "write raw tables in dv2" diff from the "generate dv2 final tables" diff, and this was the easiest way to get the TypingDeduping tests passing with just raw tables implemented.

(I'm planning to release this + https://github.com/airbytehq/airbyte/pull/36936 together)